packages/ui/Scroller/Scroller.js

```diff
@@ -53,7 +53,13 @@ const scrollerDefaultProps = {
 const Scroller = (props) => {
 	// Hooks
 
-	const scrollerProps = Object.assign({}, scrollerDefaultProps, props);
+	const scrollerProps = Object.assign({}, props);
+	for (const prop in scrollerDefaultProps) {
+		// eslint-disable-next-line no-undefined
+		if (scrollerProps[prop] === undefined) {
```

Comment:
Don't we need general check for the prop like `if (!scrollerProps[prop])` ?  
Is it OK to check only `=== undefined` ?

Reply:
I checked React's defaultProps for falsy values(null, undefined, false, NaN, 0, -0, "") and found that it sets the prop value to its default value only when `undefined` is passed. So I implemeted it to check only `undefined` case. I will add this to Additional Considerations sections.